src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/HiddenFieldCheck.java

```diff
@@ -341,7 +341,6 @@ private void processVariable(DetailAST ast) {
             // local variable or parameter. Does it shadow a field?
             final DetailAST nameAST = ast.findFirstToken(TokenTypes.IDENT);
             final String name = nameAST.getText();
-
```

Comment:
Please restore this empty line.
It is not related to target fix.

Reply:
done